modules/tinymce/src/themes/silver/main/ts/api/Settings.ts

```diff
@@ -50,6 +51,25 @@ const isMergeStyleFormats = (editor: Editor): boolean => editor.getParam('style_
 const getLineHeightFormats = (editor: Editor): string[] =>
   editor.getParam('lineheight_formats', '1 1.1 1.2 1.3 1.4 1.5 2', 'string').split(' ');
 
+const getContentLangugaes = (editor: Editor): ContentLanguage[] => {
+  const defaultSettings = [
+    { code: 'en', title: 'English (US)' },
```

Comment:
Can we please swap the order of these, as this goes against existing code as the `title` is always first generally.